draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -497,6 +545,15 @@ The code presented here corresponds to the example Sage {{SAGE}} code found at
 {{github-repo}}. Which is additionally used to generate intermediate test
 vectors. The Sage code is also checked against the hacspec implementation.
 
+Note that each encoding requires that certain preconditions must hold in
+order to be applied.
+
+## Encodings for Weierstrass curves
```

Comment:
What is the purpose of this section? Do we plan to specify encodings for other curve representations?

Reply:
Basically, I was planning to include encodings for 
 2. Montgomery curves
 2.1. Elligator2 ( A!=0 )
 2.2  Elligator2 specillized for  A=0 

However, this can be discussed in the future.